src/config/env.validation.ts

```diff
@@ -32,7 +32,7 @@ class EnvironmentVariables {
   DB_USERNAME: string;
 
   @IsString()
-  @IsNotEmpty()
+  // @IsNotEmpty()
```

Comment:
if not used, could directly remove it

Reply:
Oh..This is my bad. My development environment has no password by default.

Reply:
Revised.
[fix: restore proper authentication](https://github.com/Cophr/backend/pull/10/commits/067c46e4a8441436a54e236bb1e78f52e5989d5f)